core/models/event.py

```diff
@@ -272,6 +272,26 @@ def set_random_cover(self):
             # No StockPicture available
             pass
 
+    def clone(self):
+        """
+        Clone this Event, for events with multiple website - that is English and non-English version.
+
+        :return: the cloned Event
+        :rtype: Event or NoneType
+        """
+        if not self.id:
+            # this Event isn't saved yet...
+            return None
+
+        # first, get a copy of self (that isn't self), we'll save this into
+        # the database as a new record by wiping the ID and re-saving
+        clone = Event.objects.get(id=self.id)
+        clone.id = None
+        clone.name += ' (clone)'
+        clone.page_url += '_(clone)'
```

Comment:
@amakarudze if this is a url we probably don't want brackets in as they're not valid characters in a url.